src/App.js

```diff
@@ -1,10 +1,56 @@
 import Background from "./assets/Background.svg";
 import Logo from "./assets/spacecenter1.svg";
+import { useNavigate } from "react-router-dom";
+
+async function post(url, body) {
+  const response = await fetch(process.env.REACT_APP_API_URL + url, {
+    method: "POST",
+    body: JSON.stringify(body),
+    headers: {
+      "Content-type": "application/json; charset=UTF-8",
+    },
+  });
+  const json = await response.json();
+  return json;
+}
 
 function App() {
+  const navigate = useNavigate();
+
+  const handleSubmit = async () => {
+    const form = document.forms["register"];
```

Comment:
It's better to use FormData here. The `handleSubmit` function receives an event from the form, and you can get the data from there.
```
const handleSubmit = (event) => {
  event.preventDefault() // <- prevents the page reloading
  const data = new FormData(event.target);
  const mail = data.get("email")
  const pass = data.get("password")
  ...
```
This requires the form inputs to have a `name` attribute instead of id

Reply:
Oh ok, thanks! I misunderstood